// try{

//     console.log(eval("1+2+7#5"));
// }

// catch(err){

//     console.log("error occured");
// }

try{

    console.log(eval("1+2+7#5"));
}

catch(err){

    console.log(eval("1+2+7#5"));
}

finally{

    console.log("api call");

    console.log("populate html pages");
}